components/menu/index.tsx

```diff
@@ -38,68 +33,37 @@ const Menu: FC = observer(() => {
   return (
     <>
       <div className="flex flex-col">
-        <MenuItemGroup
-          groupName="markets"
-          hideLabel={hideLabels}
-          className="mt-ztg-32 mb-ztg-12"
-        />
-        <MenuItem
-          href={createItem.href}
-          IconComponent={createItem.IconComponent}
-          hideLabel={hideLabels}
-          active={navigationStore.checkPage("create")}
-          className="mb-ztg-12"
-          onClick={() => {
-            navigate("create");
-          }}
-        >
-          {createItem.label}
-        </MenuItem>
-        <MenuItemGroup
-          groupName="account"
-          hideLabel={hideLabels}
-          className="mb-ztg-12"
-        />
-        <MenuItem
-          href={liquidityItem.href}
-          IconComponent={liquidityItem.IconComponent}
-          hideLabel={hideLabels}
-          active={navigationStore.checkPage(liquidityItem.pageName)}
-          className="mb-ztg-12"
-          onClick={() => {
-            navigate(liquidityItem.pageName);
-          }}
-        >
-          {liquidityItem.label}
-        </MenuItem>
-        {process.env.NEXT_PUBLIC_SHOW_COURT === "true" ? (
-          <MenuItem
-            href={courtItem.href}
-            IconComponent={courtItem.IconComponent}
-            hideLabel={hideLabels}
-            active={navigationStore.checkPage(courtItem.pageName)}
-            className="mb-ztg-12"
-            onClick={() => {
-              navigate(courtItem.pageName);
-            }}
-          >
-            {courtItem.label}
-          </MenuItem>
-        ) : (
-          <></>
-        )}
-        {/* <MenuItem
-          href={activityItem.href}
-          IconComponent={activityItem.IconComponent}
-          hideLabel={hideLabels}
-          active={navigationStore.checkPage(activityItem.pageName)}
-          className="mb-ztg-12"
-          onClick={() => {
-            navigate(activityItem.pageName);
-          }}
-        >
-          {activityItem.label}
-        </MenuItem> */}
+        {
+          Object.keys(navigationStore.items).map((itemKey, idx) => {
+            const item = navigationStore.items[itemKey];
+
+            // The top item should have a marginTop property.
+            const className = idx === 0 ? "mt-ztg-32 mb-ztg-12" : "mb-ztg-12";
```

Comment:
Think it's simpler to have `mb-ztg-12` as a class on `MenuItem` and `mb-ztg-32` on the div above

Reply:
fixed [289be98](https://github.com/zeitgeistpm/ui/pull/441/commits/289be9872641755e66e2cba16adf978547d967a8)